python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -229,6 +233,100 @@ def __startPauseClicked( self, button ) :
 	def __stopClicked( self, button ) :
 		self.getPlug().setValue( GafferScene.InteractiveRender.State.Stopped )
 
+###############################################################################
+# A widget presenting a summary of the messages in a render nodes messages plug
+###############################################################################
+
+class _MessageSummaryPlugValueWidget( GafferUI.PlugValueWidget ) :
+
+	def __init__( self, plug ) :
+
+		self.__summaryWidget = GafferUI.MessageSummaryWidget( criticalOnly = True, buttonToolTip = "Click to open the render log" )
+		GafferUI.PlugValueWidget.__init__( self, self.__summaryWidget, plug )
+
+		self.__summaryWidget.levelButtonClickedSignal().connect( Gaffer.WeakMethod( self.__levelButtonClicked ), scoped = False )
+
+		self._updateFromPlug()
+
+	def getToolTip( self ) :
+
+		return None
```

Comment:
I think elsewhere there's an assumption that tooltips are strings, so "" might be more appropriate. Is this to avoid getting awkward tooltips in between the cracks of the buttons? If so (or if not) it would be good to document the reason.

While we're talking about tooltips, it'd be great to have the name of the render node in a tooltip somewhere. I was just playing with a catalogue and several render nodes and wasn't always sure what node I was about to restart. Doesn't need to be in this PR, but would be nice to do.

Reply:
https://github.com/GafferHQ/gaffer/pull/3805/commits/53be6b5a73f7b50711d28729ef0cede9085dd850#diff-025a5d68e8d612161cc57c7ca80cea1bL663-R679 and https://github.com/GafferHQ/gaffer/pull/3805/commits/b65911def743f09c48019dd076b24fa3d69da889#diff-6337748cd751cfa6969b07c0aad7d52eL257-R363